pgactivity/views.py

```diff
@@ -12,7 +12,10 @@
 from . import colors, utils
 from .activities import sorted as sorted_processes
 from .compat import link
-from .keys import BINDINGS, EXIT_KEY
+from .keys import (
+    BINDINGS,
+    EXIT_KEY,
+)
```

Comment:
This hunk is unrelated, can you revert it?

Reply:
It was requested by isort.

Reply:
```
pg_activity on  xmin_425 [$?⇕] via 🐍 v3.13.2 (.venv)
❯ isort ./pgactivity/
Fixing /home/benoit/git/dalibo/pg_activity/pgactivity/views.py

pg_activity on  xmin_425 [$!?⇕] via 🐍 v3.13.2 (.venv)
❯ git diff
diff --git a/pgactivity/views.py b/pgactivity/views.py
index b8720af..190905a 100644
--- a/pgactivity/views.py
+++ b/pgactivity/views.py
@@ -12,7 +12,10 @@ from blessed import Terminal
 from . import colors, utils
 from .activities import sorted as sorted_processes
 from .compat import link
-from .keys import BINDINGS, EXIT_KEY
+from .keys import (
+    BINDINGS,
+    EXIT_KEY,
+)
 from .keys import HELP as HELP_KEY
 from .keys import (
     KEYS_BY_QUERYMODE,
```

Reply:
It's in a separate commit.